.gitignore

```diff
@@ -29,3 +29,6 @@ yarn-error.log*
 
 # Snapshot
 blockchain/
+
+# Ouputs
+outputs.ts
```

Comment:
Do you mean deployments/outputs.ts?

Reply:
Not sure if the path makes a difference but added it for granularity's sake